Services/housekeeping_service.c

```diff
@@ -1,121 +1,100 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file housekeeping_service.c
+ * @author Haoran Qi, Andrew Rooney
+ * @date 2020-07-07
+ */
+#include <FreeRTOS.h>
+#include <csp/csp.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <stdlib.h>
-
-#include <FreeRTOS.h>
-#include <csp/csp.h>
-#include <csp/arch/csp_thread.h>
-#include <csp/drivers/usart.h>
-#include <csp/drivers/can_socketcan.h>
-#include <csp/interfaces/csp_if_zmqhub.h>
 
 #include "housekeeping_service.h"
+#include "system.h"
+#include "service_utilities.h"
 #include "services.h"
-#include "demo.h"
 
+extern Service_Queues_t service_queues;
 unsigned int count = 0;
-extern service_queues_t service_queues;
-
 
 SAT_returnState hk_service_app(csp_packet_t *pkt) {
- printf("Test HK Service 1\n");
   uint8_t ser_subtype = (uint8_t)pkt->data[0];
- printf("Test HK Service 2\n");
+  ex2_log("Test HK Service 2\n");
   switch (ser_subtype) {
-    case HK_PARAMETERS_REPORT: 
-			printf("Test HK Service 3\n");
-	    	pkt = tc_hk_para_rep(pkt);
-			printf("Test HK Service 4\n");
-		    if(pkt->data[1]!= NULL){
-		    		csp_log_info("HK_REPORT_PARAMETERS TASK FINISHED");
-			  }
-			sleep(1);
-			printf("Ground Station Task Checkout\n");
-		  	if(pkt->data[0] == TM_HK_PARAMETERS_REPORT){//determine if needed to send back to ground
-				  printf("Enter Loop\n");
-	  		 	  ground_response_task(pkt);
-				  printf("Loop Finished\n");
-			  }
-	  		break;	    
+    case HK_PARAMETERS_REPORT:
+      pkt = tc_hk_para_rep(pkt);
+      if (pkt->data[1] != NULL) {
+        csp_log_info("HK_REPORT_PARAMETERS TASK FINISHED");
+			}
+      ex2_log("Ground Station Task Checkout\n");
+      if (pkt->data[0] ==
+          TM_HK_PARAMETERS_REPORT) {  // determine if needed
+                                      // to send back to ground
+        ex2_log("Enter Loop\n");
+        ground_response_task(pkt);
+        ex2_log("Loop Finished\n");
+      }
+      break;
+
     default:
-	    	csp_log_error("HK SERVICE NOT FOUND SUBTASK");
-	    	csp_buffer_free(pkt);
-	    	return SATR_ERROR;
-	}
+      csp_log_error("HK SERVICE NOT FOUND SUBTASK");
+      csp_buffer_free(pkt);
+      return SATR_ERROR;
+  }
```

Comment:
Is it decided that we're indenting on 2s?

Reply:
I'll have to go back to the Google style guide. I can't remember. I'll maybe just run clang-format again

Reply:
Oh sorry for that I think it's my fault. When I wrote all those things I didn't really care the style much. So how many wo should indent in usual style?

Reply:
All good. Yes, 2 spaces according to Google style https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs

Reply:
For future reference, please run `git-clang-format` before committing your code and it will fix these errors for you.